datalad/crawler/nodes/tests/test_annex.py

```diff
@@ -25,6 +26,22 @@
 from ....support.stats import ActivityStats
 from ....support.annexrepo import AnnexRepo
 
+
+@with_tempfile(mkdir=True)
+def test_annexificator_no_git_if_dirty(outdir):
+
+    eq_(listdir(outdir), [])
+
+    filepath = opj(outdir, 'myfile.txt')
```

Comment:
if I were you I would have generated `.something` so to make sure that noone is left unattended